.github/actions/tests-module-moordyn/action.yml

```diff
@@ -0,0 +1,25 @@
+name: 'MoorDyn module tests'
+description: 'Run tests specific to the MoorDyn module'
+author: 'Andy Platt https://github.com/andrew-platt'
+
+
+inputs:
+  test-target:
+    description: 'Which tests to run: unit | regression | all'
+    default: 'all'
+
+runs:
+  using: "composite"
+  steps: 
+    - run: |
+
+        if [[ ${{ inputs.test-target }} == "unit" ]] || [[ ${{ inputs.test-target }} == "all" ]]; then
+          ctest -VV -R fvw_utest
```

Comment:
This should probably be something related to MoorDyn and not fvw

Reply:
oops.  I'll fix that.